types/react-sizes/react-sizes-tests.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { WithSizes, Sizes } from 'react-sizes';
```

Comment:
If `default` is set, you can change this import to: `import WithSizes, { Sizes } from 'react-sizes';`

Reply:
thanks for the review, will test this out

Reply:
it looks like the `react-sizes` exports `withSizes` as both a named export and the default export

![Screen Shot 2019-07-09 at 12 21 35 AM](https://user-images.githubusercontent.com/2119400/60867397-96207800-a1df-11e9-8396-137c5351afdd.png)


https://github.com/renatorib/react-sizes/blob/master/src/index.js#L6

should we do the same for the `WithSizes` type definition? 🤔

Reply:
ok, made this change and pushed it to this branch.  tests passed locally, let's see if the CI build is green as well. 🤞

Reply:
You're right it exports both of them. Thanks for working on it.